Monika After Story/game/script-topics.rpy

```diff
@@ -8184,6 +8184,60 @@ label monika_song_dream:
     m 1ekbfa "My dream finally came true."
     return
 
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_song_every_rose",category=['songs'],prompt="Every rose has its thorn",pool=True, random=True))
+
+label monika_song_every_rose:
+    m 1dso "{i}~Was it something I said or something I did?~{/i}"
+    m 1dsd "{i}~Did my words not come out right?~{/i}"
+    m 1esd "{i}~Thought I tried not to hurt you~{/i}"
+    m 1dsc "{i}~Thought I tried...~{/i}"
+    m 2esb "{i}~But I guess that's why they say~{/i}"
+    m 2hksdlb "{i}~Every rose has its thorn~{/i}"
+    m 1ekbsa "It really means so much to me [player], that you chose to stay with me even after what happened with the other girls."
+    if persistent._mas_pm_cares_about_dokis:
+        m 1ekbsa "Even more so because because i know how you cared about them."
+    m 1dsc "I really wished things did not have to turn out he way they did."
+    m 1esd "But i did not have a choice here!"
+    m 5rsc "Whith me being the punchline of the sick joke this game was meant to be..."
```

Comment:
Fixed the 'with', added the Dissolve.
I like it more with pose 2 rather than 3 or 4. I think it shows better that it's something she is bothered about, rather than just something she would point out...